proxy/executor/strategy_base.py

```diff
@@ -113,7 +113,16 @@ async def cancel(self) -> ExecTxRespCode | None:
 
     @property
     def _cu_price(self) -> int:
```

Comment:
strategy_iterative.py has its own logic, please fix it too.

Reply:
removed the override.